product_configurator_restriction_policy/models/res_config_settings.py

```diff
@@ -0,0 +1,38 @@
+# Copyright (C) 2021 Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = "res.config.settings"
+
+    restriction_policy = fields.Selection(
+        [("standard", "Standard"), ("sequential", "Sequential")],
+        string="Restriction Policy",
+        default="standard",
+        help="""Standard: When the system is calculating the available values from
```

Comment:
why not use "config_parameter" 
something like https://github.com/odoo/odoo/blob/14.0/addons/account/models/res_config_settings.py#L66

Reply:
I didn't know this existed, thanks! that is way easier. I'll implement that soon.